contracts/token/ERC20WithPermit.sol

```diff
@@ -34,9 +34,10 @@ contract ERC20WithPermit is IERC20WithPermit, Ownable {
 
     /// @notice Returns EIP2612 Permit message hash. Used to construct EIP2612
     ///         signature provided to `permit` function.
-    // keccak256("Permit(address owner,address spender,uint256 value,uint256 nonce,uint256 deadline)");
     bytes32 public constant override PERMIT_TYPEHASH =
-        0x6e71edae12b1b97f4d1f60370fef10105fa2faae0126114a169c64845d6126c9;
+        keccak256(
```

Comment:
Can we include in the test the expected value that used to be hardcoded?

Reply:
Done